src/components/RiskComponents.vue

```diff
@@ -5,7 +5,7 @@
       <v-col
         v-for="(risk, index) in riskData"
         :key="risk.type"
-        cols="3"
+        cols="2"
```

Comment:
Have you tried `2.5`? 
<img width="1666" alt="Screen Shot 2020-11-21 at 5 14 53 PM" src="https://user-images.githubusercontent.com/12633533/99891292-1a309f80-2c1d-11eb-9942-f91cb3910dbd.png">

or `2.75`
<img width="1666" alt="Screen Shot 2020-11-21 at 5 15 57 PM" src="https://user-images.githubusercontent.com/12633533/99891305-42200300-2c1d-11eb-840d-316713157daf.png">